src/main/java/competition/subsystems/armlifting/CollectorArmLiftingSubsystem.java

```diff
@@ -51,9 +59,15 @@ public boolean isAtMinimum() {
     } 
     
     public void setPower (double power) {
+
+        if(contract.isCollectorArmLiftingReady()) {
+            liftingCollectorArmMotor.simpleSet(power);
+        }
+
```

Comment:
This needs to be after all the checks for maximum/minimum power, or it won't work.